include/gatekeeper_gk.h

```diff
@@ -57,6 +57,26 @@ struct gk_instance {
 	uint16_t          rx_queue_back;
 	/* TX queue on the back interface. */
 	uint16_t          tx_queue_back;
+	/* Total number of packets received. */
+	uint64_t          tot_pkts_num;
+	/* Total size of packets received. */
+	uint64_t          tot_pkts_size;
```

Comment:
For each of the `size` variables I think we should mention in the comment that they are in bytes.